src/Ng/CommandHelpers.cs

```diff
@@ -140,11 +236,7 @@ public static bool GetVerbose(IDictionary<string, string> arguments)
         public static bool GetBool(IDictionary<string, string> arguments, string argumentName, bool defaultValue)
         {
             string argumentValue;
-            if (!arguments.TryGetValue("-" + argumentName, out argumentValue))
-            {
-                return defaultValue;
-            }
-
+            TryGetArgument(arguments, argumentName, out argumentValue);
             return string.IsNullOrEmpty(argumentValue) ? defaultValue : argumentValue.Equals("true", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
yes

---
In reply to: [71233002](https://github.com/NuGet/NuGet.Services.Metadata/pull/77#discussion_r71233002) [](ancestors = 71233002,71022687)